NotifyMe/views/views.py

```diff
@@ -0,0 +1,248 @@
+from django.shortcuts import render
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from ..serializers import UserSerializer, SubscriptionSerializer, NotificationSerializer
+from NotifyMe.models.user import User
+from NotifyMe.models.notification import Notification
+from NotifyMe.models.notificationType import NotificationType
+from NotifyMe.models.subscription import Subscription
+from NotifyMe.models.subscriptionPlan import SubscriptionPlan
+from rest_framework import status 
+from rest_framework.views import APIView
+from NotifyMe.services.service import *
+
+
+# ---------USER-API---------------- 
+# DONOT CALL MODEL INSIDE VIEW---> ONLY CALL SERVICE AND SERIALIZER
+
+class UserAPI(APIView, UserService):
+  
+  def get(self, request):
+    try:
+      objects = self.get_user_data(request)
+      serializer = UserSerializer(objects, many=True)
+      return Response(serializer.data)
+    except Exception as e:
+      print(f"Error: {e}")
+      return Response({"error": str(e)}, status=status.HTTP_500_INTERNAL_SERVER_ERROR)
```

Comment:
define fixed response type in all the api response refer any article to get response structure